setup.py

```diff
@@ -12,10 +14,8 @@ def readme():
       url='http://github.com/broadinstitute/herc',
       author='The Broad Institute',
       packages=find_packages(exclude='tests'),
-      package_data={
-          # Include everything in data/, both schemas and examples.
-          '': ['data/*']
-      },
+      data_files=[('data/aurora', ['data/aurora/api.thrift']),
+                  ('data/schemas', ['data/schemas/jobsubmit.json'])],
```

Comment:
That's upsetting. There's more in the data/ dir that should be packaged, though. I suppose you could specify everything manually but something in [this big mess of words](https://pythonhosted.org/setuptools/setuptools.html#including-data-files) should be what you need.